packages/beacon-node/src/network/peers/score/store.ts

```diff
@@ -55,8 +57,10 @@ export class PeerRpcScoreStore implements IPeerRpcScoreStore {
 
   applyAction(peer: PeerId, action: PeerAction, actionName: string): void {
     const peerScore = this.scores.getOrDefault(peer.toString());
-    peerScore.add(peerActionScore[action]);
+    const scoreChange = peerActionScore[action];
+    const newScore = peerScore.add(scoreChange);
 
+    this.logger?.debug("peer score adjusted", {scoreChange, newScore, peerId: peer.toString(), actionName});
```

Comment:
should we use `prettyPrintPeerId` to make it consistent to other logs?

Reply:
Good catch!  I got one place but just updated the one you saw!